### PR TITLE
Upgrade Ozone ufs to 1.0.0 version

### LINKED ISSUE #12202: Upgrade Ozone from 0.5.0 to new latest release version
Now alluxio can support Ozone as an Under Storage, but the release version is not a GA release version, so this issue aim to upgrade Ozone to a latest GA release version.

### PR DESCRIPTION
**Pull request by maobaolong**
Fix #12202 #12338

### CODE REVIEW

File: shaded/ozone/pom.xml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.alluxio</groupId>
+    <artifactId>alluxio-shaded</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>alluxio-shaded-ozone</artifactId>
+  <packaging>jar</packaging>
+  <name>Alluxio Shaded Libraries - Ozone</name>
+  <version>${ufs.ozone.version}</version>
```
Comment (apc999) [inline shaded/ozone/pom.xml:23]:
```suggestion
  <version>2.5.0-SNAPSHOT</version>
```

Having a var here `${ufs.ozone.version}` is not encouraged

File: underfs/ozone/pom.xml [hunk 2]
```diff
@@ -25,17 +25,15 @@
...
     <dependency>
-      <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-ozone-client</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-ozone-filesystem</artifactId>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-shaded-ozone</artifactId>
+      <version>${ufs.ozone.version}</version>
```
Comment (apc999) [inline underfs/ozone/pom.xml:36]:
```suggestion
      <version>${project.version}</version>
```

**Review by apc999 — APPROVED** [review]
Thanks for fixing this!

**Comment by maobaolong** [comment]
@apc999 Thank you for the review and help.

**Review by apc999 — APPROVED** [review]
LGTM. thanks the fix!

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**